### ISSUE TITLE
Hypothesis: there are politicians spending mostly with people from the same political party

### ISSUE THREAD

**Issue by cuducos** [datasciencebr/serenata-de-amor#121]
We need [this database](http://dados.gov.br/dataset/filiados-partidos-politicos) and #16 — does that make sense?

**Comment by anaschwendler** [comment]
In short, what @cuducos was trying to say is that the hypothesis focus in finding if all the congressperson spend money always in the same places, and checking if those places belong to people from the same party.

**Comment by luizfzs** [comment]
I think it is a good analysis, that may yield some interesting results. In the cases that there are some spending pattern, we could come with a more detailed analysis to explore deeper. I've got more details on my mind I'm not sure whether this is the place to discuss them.

**Comment by cuducos** [comment]
Yep, this is the place! If what you have in mind is related to that hypothesis, you're welcome here; otherwise you're welcome to start a new issue about your ideas too.

**Comment by dehatanes** [comment]
Hello :)
I was taking a look in the github issues and noticed this one that seems pretty cool but don't have any comments since a long time ago.

I want to explore this hypothesis but I am new here and I am afraid that I missed something... do we have further work on this issue that I can find in another place and was not referenced here? (I didn't find anything in the notebooks that I saw)

**Comment by dehatanes** [comment]
I forgot to mention here, but I started to look into this issue in this repository, if anyone is interested 😄  https://github.com/dehatanes/CEAP-reimbursement-analysis

I am leaving this comment here because I intend to upload the notebook in the serenata's repository in the future, but I am afraid I will not have much time to work with it in a short run, so it may take a while until I format the work to open a PR.
